tools/devctr/Dockerfile.x86_64

```diff
@@ -89,6 +89,14 @@ RUN mkdir "$TMP_BUILD_DIR" \
     && cd / \
     && rm -rf "$TMP_BUILD_DIR"
 
+# Build iperf3-vsock
+RUN mkdir "TMP_BUILD_DIR" && cd "TMP_BUILD_DIR" \
+    && git clone https://github.com/stefano-garzarella/iperf-vsock && cd iperf-vsock \
```

Comment:
Can we stick to a release, tag or commit hash? Otherwise we introduce variability from container version to container version that is not obvious.

Reply:
makes sense, I will modify this

Reply:
done